src/Tizen.NUI/src/public/WebView/WebBackForwardList.cs

```diff
@@ -36,9 +35,8 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         }
 
         /// <summary>
-        /// Get uri.
+        /// Get url.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public string Url
```

Comment:
We need tizen version tag here.
And also all other public apis too.

Reply:
Done.